github/readmes.md

```diff
@@ -0,0 +1,9 @@
+# Project Readme and Description
+
+Projects must have a description and a readme that explains what it is for and where it fits in to the big picture.
+
+Some indication of what other services it collaborates with, what kind of storage, data structures or CRD's it depends on would be good to have in the README as well.
+
+Depending on how 'far up the stack' they are, they can have usage examples.
+
+If it is a tool that we make public, then usage examples are a must.
```

Comment:
You have an example for the usage example? I would not know how to write that now.

Reply:
@xh3b4sd added some text, does that help?

Reply:
Yes that is cool. Thank you.